### ISSUE TITLE
Settings not saved

### ISSUE THREAD

**Issue by rubendel** [opensourceBIM/ModeltoolsPlugins#4]
Version 0.4

Initially there are 6 profiles already. When I try to add another profile and restart Sketchup (15.3.331 64bit, Windows), the added profile is gone. The same for deleting profiles. Changing the name of an existing profile does not change the name in the dropdown box.

**Comment by rubendel** [comment]
```
Error: #<TypeError: no implicit conversion of nil into String>
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/BIMserver/profiles.rb:91:in `initialize'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/BIMserver/profiles.rb:91:in `open'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/BIMserver/profiles.rb:91:in `write_config'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/BIMserver/profile_window.rb:157:in `block in initialize'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/events.rb:82:in `call'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/events.rb:82:in `block in trigger_event'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/events.rb:78:in `each'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/events.rb:78:in `trigger_event'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/window.rb:355:in `event_control_callback'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/window.rb:310:in `callback_handler'
C:/Users/Ruben de Laat/AppData/Roaming/SketchUp/SketchUp 2015/SketchUp/Plugins/opensourceBIM/lib/SKUI/window.rb:268:in `block in add_callback'
SketchUp:1:in `call'
```

**Comment by janbrouwer** [comment]
I think the issue is resolved in version 0.4.2
